src/GeoFabrics/dem.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jun 18 10:52:49 2021
+
+@author: pearsonra
+"""
+import rioxarray
+import numpy
+import scipy.interpolate 
+
+class ReferenceDem:
+    """ A class to manage the reference DEM in a catchment context
+    
+    Specifically, clip within the catchment and outside any LiDAR. If
+    set_foreshore is True set all positive DEM values in the foreshore to zero
+    """
+    
+    def __init__(self, dem_file, catchment_geometry, set_foreshore = True):
+        """ Load in dem """
+        
+        self.catchment_geometry = catchment_geometry
+        self.set_foreshore = set_foreshore
+        self._dem = rioxarray.rioxarray.open_rasterio(dem_file, masked=True)
+        
+        self.__set_up()
```

Comment:
I'm unsure of the use of a double underscore here, compared to a single one. I'm not telling you to change it but just highlighting it

Reply:
I did some reading (link to docs below), which indicates that a double underscore should be used if you want to do name-mangling and a single underscore should be used for private variables - which is what I'm after so I'll change it thanks! 

https://docs.python.org/3/tutorial/classes.html#private-variables